### ISSUE TITLE
Conflict with Token Magic FX

### ISSUE THREAD

**Issue by DivertedCircle** [dev7355608/perfect-vision#94]
Since the 2.0.0 update having Perfect Vision Enabled along with Token Magic FX is resulting in my maps not loading at all and the following error being thrown.

```
09:43:56.144 Uncaught TypeError: this.placeableImg.parent.data is undefined
    handleTransform FilterZoomBlur.js:24
    apply FilterProto.js:106
    pop filter-system.js:151
    renderAdvanced display.min.js:8
    render display.min.js:8
    render display.min.js:8
    render display.min.js:8
    renderAdvanced display.min.js:8
    render display.min.js:8
    renderAdvanced display.min.js:8
    render display.min.js:8
    render core.min.js:8
    render app.min.js:8
    emit ticker.min.js:8
    update ticker.min.js:8
    _tick ticker.min.js:8
FilterZoomBlur.js:24:1
    handleTransform FilterZoomBlur.js:24
    apply FilterProto.js:106
    pop filter-system.js:151
    renderAdvanced display.min.js:8
    render display.min.js:8
    render display.min.js:8
    render display.min.js:8
    renderAdvanced display.min.js:8
    render display.min.js:8
    renderAdvanced display.min.js:8
    render display.min.js:8
    render core.min.js:8
    render app.min.js:8
    emit ticker.min.js:8
    update ticker.min.js:8
    _tick ticker.min.js:8
```

**Comment by dev7355608** [comment]
This issue will be resolved by the next Token Magic FX update. Coming soon.

**Comment by dev7355608** [comment]
The issue is also fixed in PV 2.3.0.
